Documentation November quick wins (#261)
opendatasoft/ods-documentation: source/publishing_data/01_creating_a_dataset/sourcing_data.rst

.. admonition:: Important
   :class: important

>    When synchronizing from a remote server, Opendatasoft keeps a persistent cache and does not automatically prune files missing from the remote directory. Please use the "Clean cache" button on the ressource if you need some cleanup to be performed.

Reviewer: Same comments as above.
```suggestion
   When synchronizing from a remote server, Opendatasoft keeps a persistent cache and does not automatically prune files missing from the remote directory. If you need some cleanup to be performed, to the right of the resource, click "Clean cache".
```
